apis/compute/v1beta1/zz_windowsvirtualmachinescaleset_types.go

```diff
@@ -680,6 +680,7 @@ type WindowsVirtualMachineScaleSetObservation struct {
 	// A scale_in block as defined below.
 	ScaleIn []WindowsVirtualMachineScaleSetScaleInObservation `json:"scaleIn,omitempty" tf:"scale_in,omitempty"`
 
+	// Deprecated, scale_in_policy will be removed in favour of the scale_in code block.
```

Comment:
```suggestion
	// Deprecated: ...
```

Please also see: https://github.com/golang/go/wiki/Deprecated